controllers/tweetController.js

```diff
@@ -69,12 +73,49 @@ const tweetController = {
               return result
             }, {}),
           isLiked: tweet.Likes.map(l => l.UserId).includes(helpers.getUser(req).id),
-          repliesCount: tweet.Replies.length,
-          likesCount: tweet.Likes.length
+          repliedCount: tweet.Replies.length,
+          LikeCount: tweet.Likes.length,
+          isSelf: tweet.UserId === helpers.getUser(req).id
         }
-
         return res.json(tweet)
       })
+      .catch(err => next(err))
+  },
+  updateTweet: (req, res, next) => {
+    const { description } = req.body
+    Tweet.findByPk(req.params.id)
+      .then(tweet => {
+        if (!tweet) {
+          return res.status(409).json({ status: 'failure', message: 'tweet not exist' })
+        }
+        if (tweet.UserId !== helpers.getUser(req).id) {
+          return res.status(409).json({ status: 'failure', message: 'permission denied' })
+        }
```

Comment:
這裡屬於未被授權，應為 [401 Unauthorized](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/401)
> The HTTP 401 Unauthorized client error status response code indicates that the request has not been applied because it lacks valid authentication credentials for the target resource.